sympy/physics/control/lti.py

```diff
@@ -76,6 +77,53 @@ def bilinear(tf, sample_per):
         return num_coefs, den_coefs
 
 
+def backward_diff(tf, sample_per):
+
+        """
+
+        Returns falling coefficients of H(z) from numerator and denominator.
+        Where H(z) is the corresponding discretized transfer function,
+        discretized with the backward difference transform method.
+        H(z) is obtained from the continuous transfer function H(s)
+        by substituting s(z) =  (z-1)/(T*z) into H(s), where T is the
+        sample period.
+        H(z) corresponds to the difference equation of the by backward difference scheme.
```

Comment:
`of the by backward difference scheme` doesn't seem to make sense. Should it simply be `of the backward difference scheme` ?

Reply:
👍🏽

Reply:
> `of the by backward difference scheme` doesn't seem to make sense. Should it simply be `of the backward difference scheme` ?

-> corrected